go3/settings.py

```diff
@@ -278,6 +278,7 @@
 SENDGRID_API_KEY = env('SENDGRID_API_KEY', default='456')
 SENDGRID_SANDBOX_MODE_IN_DEBUG = env('SENDGRID_SANDBOX_MODE_IN_DEBUG', default=True)
 SENDGRID_TRACK_CLICKS_HTML = False
+EMAIL_DISABLE = False
```

Comment:
For safety, how about we do this in the affirmative and make it configurable via environment?

```suggestion
EMAIL_ENABLE = env("EMAIL_ENABLE", default=False)
```

If you accept this, we also need to add it to the list of recognized environment variables on line 38